mvg/features/modeid.py

```diff
@@ -28,7 +28,9 @@ def __init__(self, results, t_zone=None, t_unit=None):
         else:
             dict_for_df = self.results().copy()
             dict_for_emerging = dict_for_df.pop("mode_info")
+            dict_for_mode_proba = dict_for_df.pop("", {})
```

Comment:
Why empty key? Should be mode_probabilities I guedd.

Reply:
That is a fantastic point